Makefile.am

```diff
@@ -32,8 +32,8 @@ SUBDIRS = libcompat \
 	libcfnet \
 	libenv \
 	libpromises \
-	cf-agent \
 	cf-check \
+	cf-agent \
```

Comment:
Is this change a dependency sort of change? Like cf-check must be built before cf-agent?

Reply:
It was, `libcf-check.la` was compiled inside `cf-check` and then used for `cf-agent`. However, since I switched to including `cf-check` in `libpromises` instead, it's no longer necessary.